src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChange = (movieProperty: string) => {
```

Comment:
Move this from the App, bc you create this function each render
Split your ` movieProperty.toLowerCase().includes(query.toLowerCase().trim());` with variables
```suggestion
  const IsIncludes = (string: string, substring: string) => {
```

Reply:
I'm sorry, I don't understand how to fix this, as you said (

Reply:
How to move from the component?
Or how to split your looong `movieProperty.toLowerCase().includes(query.toLowerCase().trim())`  with the variables?
```js
 const isIncludes = (string: string, substring: string) => {
    const normalizedString = normalizedString.toLowerCase();
    const normalizedSubstring = query.toLowerCase().trim();

    return normalizedString.includes(normalizedSubstring);
}
```